subprojects/packagefiles/taglib/meson.build

```diff
@@ -0,0 +1,316 @@
+  'taglib/mpc/mpcproperties.cpp',
+  'taglib/mpeg/mpegfile.cpp',
+  'taglib/mpeg/mpegheader.cpp',
+  'taglib/mpeg/mpegproperties.cpp',
+  'taglib/mpeg/xingheader.cpp',
+  'taglib/mpeg/id3v1/id3v1genres.cpp',
+  'taglib/mpeg/id3v1/id3v1tag.cpp',
+  'taglib/mpeg/id3v2/id3v2extendedheader.cpp',
+  'taglib/mpeg/id3v2/id3v2footer.cpp',
+  'taglib/mpeg/id3v2/id3v2frame.cpp',
+  'taglib/mpeg/id3v2/id3v2framefactory.cpp',
+  'taglib/mpeg/id3v2/id3v2header.cpp',
+  'taglib/mpeg/id3v2/id3v2synchdata.cpp',
+  'taglib/mpeg/id3v2/id3v2tag.cpp',
+  'taglib/mpeg/id3v2/frames/attachedpictureframe.cpp',
+  'taglib/mpeg/id3v2/frames/chapterframe.cpp',
+  'taglib/mpeg/id3v2/frames/commentsframe.cpp',
+  'taglib/mpeg/id3v2/frames/eventtimingcodesframe.cpp',
+  'taglib/mpeg/id3v2/frames/generalencapsulatedobjectframe.cpp',
+  'taglib/mpeg/id3v2/frames/ownershipframe.cpp',
+  'taglib/mpeg/id3v2/frames/podcastframe.cpp',
+  'taglib/mpeg/id3v2/frames/popularimeterframe.cpp',
+  'taglib/mpeg/id3v2/frames/privateframe.cpp',
+  'taglib/mpeg/id3v2/frames/relativevolumeframe.cpp',
+  'taglib/mpeg/id3v2/frames/synchronizedlyricsframe.cpp',
+  'taglib/mpeg/id3v2/frames/tableofcontentsframe.cpp',
+  'taglib/mpeg/id3v2/frames/textidentificationframe.cpp',
+  'taglib/mpeg/id3v2/frames/uniquefileidentifierframe.cpp',
+  'taglib/mpeg/id3v2/frames/unknownframe.cpp',
+  'taglib/mpeg/id3v2/frames/unsynchronizedlyricsframe.cpp',
+  'taglib/mpeg/id3v2/frames/urllinkframe.cpp',
+  'taglib/ogg/oggfile.cpp',
+  'taglib/ogg/oggpage.cpp',
+  'taglib/ogg/oggpageheader.cpp',
+  'taglib/ogg/xiphcomment.cpp',
+  'taglib/ogg/flac/oggflacfile.cpp',
+  'taglib/ogg/opus/opusfile.cpp',
+  'taglib/ogg/opus/opusproperties.cpp',
+  'taglib/ogg/speex/speexfile.cpp',
+  'taglib/ogg/speex/speexproperties.cpp',
+  'taglib/ogg/vorbis/vorbisfile.cpp',
+  'taglib/ogg/vorbis/vorbisproperties.cpp',
+  'taglib/riff/rifffile.cpp',
+  'taglib/riff/aiff/aifffile.cpp',
+  'taglib/riff/aiff/aiffproperties.cpp',
+  'taglib/riff/wav/infotag.cpp',
+  'taglib/riff/wav/wavfile.cpp',
+  'taglib/riff/wav/wavproperties.cpp',
+  'taglib/s3m/s3mfile.cpp',
+  'taglib/s3m/s3mproperties.cpp',
+  'taglib/toolkit/tbytevector.cpp',
+  'taglib/toolkit/tbytevectorlist.cpp',
+  'taglib/toolkit/tbytevectorstream.cpp',
+  'taglib/toolkit/tdebug.cpp',
+  'taglib/toolkit/tdebuglistener.cpp',
+  'taglib/toolkit/tfile.cpp',
+  'taglib/toolkit/tfilestream.cpp',
+  'taglib/toolkit/tiostream.cpp',
+  'taglib/toolkit/tpropertymap.cpp',
+  'taglib/toolkit/trefcounter.cpp',
+  'taglib/toolkit/tstring.cpp',
+  'taglib/toolkit/tstringlist.cpp',
+  'taglib/toolkit/tzlib.cpp',
+  'taglib/trueaudio/trueaudiofile.cpp',
+  'taglib/trueaudio/trueaudioproperties.cpp',
+  'taglib/wavpack/wavpackfile.cpp',
+  'taglib/wavpack/wavpackproperties.cpp',
+  'taglib/xm/xmfile.cpp',
+  'taglib/xm/xmproperties.cpp',
+  include_directories : taglib_includes,
+  install : true
+)
+
+pkg = import('pkgconfig')
+pkg.generate(taglib,
+  subdirs : 'taglib'
```

Comment:
This produces `tag.pc`, which doesn't match `taglib`. You need to set the `filebase: 'taglib'` since the default name matching the library name is wrong.